8.5-jre8/Dockerfile

```diff
@@ -24,18 +15,38 @@ ENV JAVA_OPTS -Dsite.home=/home $JAVA_OPTS
 ENV JAVA_OPTS -Dsite.tempdir=/tmp $JAVA_OPTS
 ENV JAVA_OPTS -Dport.http=80 $JAVA_OPTS
 ENV _JAVA_OPTIONS -Djava.net.preferIPv4Stack=true
-ENV AI_VERSION 2.1.0
+
+ENV PATH /usr/local/tomcat/bin:$PATH
+
+RUN rm -rf /usr/local/tomcat/webapps/
+
+COPY init_container.sh /bin/init_container.sh
+COPY web.xml /tmp/tomcat/web.xml
+COPY index.jsp /tmp/webapps/ROOT/index.jsp
+COPY background.png /tmp/webapps/ROOT/background.png
+COPY sshd_config /etc/ssh/
+COPY tmp/shared/app_insights/AI-Agent.xml /usr/local/app_insights/aiagent/
+COPY tmp/shared/app_insights/ApplicationInsights.xml /usr/local/app_insights/tomcat_lib/
 
 RUN apt-get update \
-        && apt-get install -y --no-install-recommends dialog \
-        && apt-get update \
-        && apt-get install -y --no-install-recommends openssh-server \
+        && apt-get install -y --no-install-recommends dialog openssh-server wget \
```

Comment:
I think openssh-server is already retrieved in the base image

Reply:
Right! removed.